Simplified/NYPLAnnotations.swift

```diff
@@ -142,7 +142,7 @@ final class NYPLAnnotations: NSObject {
           debugPrint(#file, "Posted Last-Read \(((parameters["target"] as! [String:Any])["selector"] as! [String:Any])["value"] as! String)")
         }
       } else {
-        guard let error = error as? NSError else { return }
+        guard let error = error as NSError? else { return }
```

Comment:
I'm delaying this to next sprint when Annotations will be fully updated to support bookmarks. The crash fix is left in as a commit, removing implicitly unwrapped optionals.